src/statistics.cpp

```diff
@@ -194,6 +194,57 @@ static stats::battle_result_map read_battle_result_map(const config& cfg)
 	return m;
 }
 
+static config write_by_cth_map(const stats::hitrate_map& m)
+{
+	config res;
+	for(const auto& i : m) {
+		const config child(
```

Comment:
You don't need to create a temp config, object here. You should be able to just do
```cpp
res.add_child("hitrate_map_entry", { 
    "cth", i.first,
    "stats", i.second.write() }
);
```
if I remember correctly.

Reply:
6e10caddfde